app/src/main/java/com/epfl/drawyourpath/challenge/Tournament.kt

```diff
@@ -0,0 +1,68 @@
+package com.epfl.drawyourpath.challenge
+
+import java.time.Clock
+import java.time.Duration
+import java.time.LocalDateTime
+
+/**
+ * class representing tournaments that the user can take part in
+ */
+data class Tournament(
+    val name: String,
+    val description: String,
+    val startDate: LocalDateTime,
+    val endDate: LocalDateTime

```

Comment:
Maybe add a list in class parameter to add all the participant

Reply:
yes we need it but we do not have a user class yet so it should be added later.